### PR TITLE
Install granite-kitchen instead of utils

### PR DESCRIPTION
**Pull request by fayvor**
#7 
- Pip install granite-kitchen instead of utils.

### CODE REVIEW

File: recipes/Getting_Started/Getting_Started_with_Replicate.ipynb [hunk 1]
```diff
@@ -136,10 +136,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
    "id": "79a5b2e1-f7eb-4de3-bfb9-f049b0c3fe10",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```
Comment (adampingel) [inline recipes/Getting_Started/Getting_Started_with_Replicate.ipynb]: Looks like we have outputs to clear here

Comment (adampingel) [reply recipes/Getting_Started/Getting_Started_with_Replicate.ipynb]: also are you handling 401 issue separately, or should we address the error here?

Comment (fayvor) [reply recipes/Getting_Started/Getting_Started_with_Replicate.ipynb]: Will clear.

Handling the 401 issue separately.

Comment (fayvor) [reply recipes/Getting_Started/Getting_Started_with_Replicate.ipynb]: Outputs cleared.

**Review by adampingel — APPROVED** [review]

**Comment by rawkintrevo** [comment]
This looks like its failling because the new library is too fat. If that is the case, then merging this will cause all tests to always fail.

**Comment by rawkintrevo** [comment]
image
looks like this has been failing (for being too fat) since last week...

**Comment by fayvor** [comment]
> This looks like its failling because the new library is too fat.

If that's the case, I would expect other projects to fail where we import `granite-kitchen` as well. Eg., why isn't `granite-code-cookbook` failing?

How much space is on the device? I wonder if there might be some import recursion going on.

**Comment by fayvor** [comment]
@rawkintrevo could we be writing too much to the log?

`System.IO.IOException: No space left on device : '/home/runner/runners/2.320.0/_diag/Worker_20241009-174642-utc.log'`

**Comment by rawkintrevo** [comment]
possible hacks:
https://github.com/marketplace/actions/maximize-build-disk-space-only-remove-unwanted-software

I found when I was looking up how much disk space the runner has

**Comment by rawkintrevo** [comment]
less hacky but cc required (cc @adampingel ) https://docs.github.com/en/actions/using-github-hosted-runners/using-larger-runners

**Comment by rawkintrevo** [comment]
Pared down example, only got rid of dotnet
https://stackoverflow.com/questions/75536771/github-runner-out-of-disk-space-after-building-docker-image

**Comment by bjhargrave** [comment]
Perhaps the kicthen is too full for individual needs? That is, instead of one very large kitchen which can make all recipes, we may need several fit-for-purpose kitchens...

**Comment by adampingel** [comment]
Yes, it was inevitable that it would have to be broken up, and we have arrived at that point.

**Comment by fayvor** [comment]
From @rawkintrevo's link:
> public shared Github runners come with around 25-29 GB of free disk

My whole virtual env is 1.5G when I run this. What else is taking up space?

If it's getting to the recipes, it would be the models.

**Comment by fayvor** [comment]
We probably don't have to pull down the 20b Ollama model in [Getting Started with Ollama:](https://github.com/ibm-granite-community/granite-kitchen/blob/7_pip_install_gk_not_utils/recipes/Getting_Started/Getting_Started_with_Ollama.ipynb)

```
Pull down Granite models:
!ollama pull granite-code:3b
!ollama pull granite-code:8b
!ollama pull granite-code:20b
```

**Comment by fayvor** [comment]
Yeah, I'm guessing the [ollama models](https://ollama.com/library/granite-code) are the problem here:
Screenshot 2024-10-09 at 2 57 16 PM

I'll put up a PR to only download the 3b.

**Pull request merged**